docs/modules/ROOT/nav.adoc

```diff
@@ -56,7 +56,7 @@ include::wan:partial$nav.adoc[]
 ** xref:osgi:design.adoc[]
 ** xref:osgi:using-osgi-service.adoc[]
 
-.Migrate to this Version
+.Upgrading From Previous Releases
```

Comment:
I like the renaming of 'migrate' to 'upgrade' 😄 

I'm not sure we need such a long label. 'Upgrade' implies replacing with a new version.

```suggestion
.Upgrade
```